iron-doc-viewer.html

```diff
@@ -318,6 +305,17 @@
               }
             }
           }
+
+          if (scrollAnchor) {
+            Polymer.dom.flush();
+            var active = this.$$('[active]');
```

Comment:
Can we change this to a classname instead? Since we use attributes/properties to configure elements, it kinda looks like `active` does something inside the elements instead of just being used to select them.